tests/message-extensions/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@tests/message-extensions",
+  "version": "0.0.0",
+  "license": "MIT",
+  "private": true,
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "scripts": {
+    "clean": "npx rimraf ./dist",
+    "build": "npx tsc",
+    "start": "node .",
+    "dev": "npx nodemon -w \"./src/**\" -e ts --exec \"node -r ts-node/register -r dotenv/config ./src/index.ts\""
+  },
+  "dependencies": {
+    "@microsoft/teams.api": "*",
+    "@microsoft/teams.apps": "*",
+    "@microsoft/teams.cards": "*",
+    "@microsoft/teams.common": "*",
+    "@microsoft/teams.dev": "*",
+    "@microsoft/teams.graph": "*"
```

Comment:
Doesn't look like graph is a used dependency for this app (same for turbo.json)

Reply:
turbo.json is used to build the sample from the `npm run build:tests` command. I've removed the graph dependency. Good catch.